nextjs-app/app/_components/Footer.tsx

```diff
@@ -18,47 +22,71 @@ export function Footer({
   heading,
   items,
 }: FooterProps) {
+  const [openIndex, setOpenIndex] = useState<number | false>(false);
+
   return (
     <footer className="bg-green py-9 md:pt-20 lg:pt-30 md:pb-20">
-        <div className="text-white">
-          <div className="page-padding">
-            <h5 className="pt-12">{heading}</h5>
-            <ul className="my-12  space-y-4">
-              {items?.length && items.map((item, i) => {
-                return(
-                  <div key={i}>
-                    <h3 className="font-serif text-24 md:text-36 lg:text-64">{item.heading}</h3>
-                    <p className="font-sans text-14 md:text-16">{item.description}</p>
-                  </div>
-                )
-              })}
-            </ul>
+      <div className="text-white">
+        <div className="page-padding">
+          <h5 className="pt-12">{heading}</h5>
+          <ul className="my-12 space-y-4">
+            {items?.length && items.map((item, i) => {
+              const isOpen = i === openIndex;
+              return(
+                <div key={i}>
+                  <h3 
+                    className="font-serif text-24 md:text-36 lg:text-64 cursor-pointer"
+                    onClick={() => setOpenIndex(isOpen ? false : i)}
+                  >
+                    {item.heading}
+                  </h3>
+                  <AnimatePresence initial={false}>
+                    {isOpen && (
+                      <motion.div
+                        initial="collapsed"
+                        animate="open"
+                        exit="collapsed"
+                        variants={{
+                          open: { opacity: 1, height: "auto" },
+                          collapsed: { opacity: 0, height: 0 }
+                        }}
```

Comment:
I think in the footer it would make sense to allow the user to have any number of accordion items open at once, since they are so short